projects/hello_world/src/hello_world.c

```diff
@@ -0,0 +1,35 @@
+//
+// Created by henry on 2021-01-09.
+//
```

Comment:
This is probably autogenerated junk from your IDE - it's okay this time, but for future code please remove it.